src/solc_0.8/polygon/child/asset/PolygonAssetV2.sol

```diff
@@ -61,11 +61,13 @@ contract PolygonAssetV2 is ERC1155ERC721 {
     /// @param ids ids to withdraw
     /// @param amounts amounts to withdraw
     function withdraw(uint256[] calldata ids, uint256[] calldata amounts) external {
+        bytes32 hash = _metadataHash[ids[0] & URI_ID];
```

Comment:
Why ```ids[0]``` ?
We should have different hash for ids

Reply:
@mvanmeerbeck Thanks for the feedback.
Just one thing though, the comments for `mint()` mention: 
`/// @param hash hash of an IPFS cidv1 folder that contains the metadata of each token type in the files: 0.json, 1.json, 2.json, etc...`

So this leads me to the assumption that the hash would be constant for that folder and the URI would be found based on `<hash>/<id>.json`

If this is not the case do let me know I'll work to resolve this issue.